geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/aggregators/HBaseArrowAggregator.scala

```diff
@@ -40,7 +40,7 @@ object HBaseArrowAggregator {
                 ecql: Option[Filter],
                 hints: Hints): (Map[String, String], FeatureReducer) = {
     val conf = ArrowScan.configure(sft, index, stats, filter, ecql, hints)
-    (conf.config + (GeoMesaCoprocessor.AggregatorClass -> classOf[HBaseArrowAggregator].getName), conf.reduce)
+    (conf.config + (GeoMesaCoprocessor.AggregatorClass -> s"$AggregatorPackage.HBaseArrowAggregator"), conf.reduce)
```

Comment:
Ok, is this 'necessary' since the class HBaseArrowAggregator may not be available at compile time?

Reply:
right, they aren't due to the split between server and client code needed to swap hbase 1 vs 2 deps